config/packages/security.yaml

```diff
@@ -1,13 +1,28 @@
 security:
+    encoders:
+        App\Entity\User:
+            algorithm: auto
```

Comment:
It's ok for this one, but you should specify an algorithm. Otherwise, if the default algorithm in Symfony changes over a new version (and you `composer update`), all your users won't be able to log in and will have to chang their passwords.

Reply:
Ok, I note that I should change that. I would do it on the connexion branch.